### PR TITLE
switch back to gofast

### PR DESCRIPTION
**Pull request by bnu0**
Unfortunately, `tomasen/fcgi_client` does not support splitting out the fcgi stderr stream from the stdout. On a well-behaved site that does not spew notices, thats fine. On most real sites, it's not fine.

### CODE REVIEW

**Comment by joshcoughlan** [comment]
Any reason not to use connection pooling? Is the benefit just not worth the added complexity?

File: runner/runner.go [hunk 1]
```diff
@@ -491,55 +492,76 @@ func runWpFpmCmdWithMetrics(subcommand []string) (string, error) {
...
+}
 
-	fcgi, err := fcgiclient.Dial(fpmUrl.Scheme, host)
+func (f *fakeHttpResponseWriter) WriteHeader(statusCode int) {
+	f.LastStatus = statusCode
+}
+
+func runWpFpmCmdSafe(subcommand []string) (string, error) {
+
+	fpmClient, err := fpm()
```
Comment (bnu0) [inline runner/runner.go:518]: Resource leak, client is Closeable!

**Comment by bnu0** [comment]
> Any reason not to use connection pooling? Is the benefit just not worth the added complexity?

If this was using TCP to a remote process, i would, but it's using a unix domain socket and the overhead of connecting to it repeatedly is probably less than any theoretical connection pool implementation.

**Review by joshcoughlan — APPROVED** [review]
LGTM

**Pull request merged**